packages/medusa/src/controllers/products/admin-list-products.ts

```diff
@@ -28,6 +28,7 @@ const listAndCount = async (
     context
```

Comment:
this method should allow to pass a transactionManager to be able to do some action as part of the parent transaction

Reply:
Not sure that this is necessary given that this will ever only include a query of data so "new writes" would never be part of an operation here?

Reply:
Fair point - maybe let's revisit as this would require a new way of thinking about the controller interface